_includes/helping.html

```diff
@@ -0,0 +1,42 @@
+<!-- Helping -->
+{% if site.locale and site.locale != "" and site.locale != nil %}
+<section class="page-section" id="{{ site.data.sitetext[site.locale].helping.section | default: "helping" }}">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-12 text-center">
+        <h2 class="section-heading text-uppercase">
+          {{ site.data.sitetext[site.locale].helping.title | markdownify | default: Helping }}</h2>
+        <h3 class="section-subheading text-muted">{{ site.data.sitetext[site.locale].helping.text | markdownify | default: "" }}</h3>
+      </div>
+    </div>
+    {% for item in site.data.sitetext[site.locale].helping.list %}
+    {% if forloop.index == 2 %}
+    <div class="row helping align-items-center text-right">
+      <div class="col-lg-6 order-md-2">
+        <a href="{{ item.url | default: assets/img/team/placeholder.jpg }}" target="_blank" rel="noopener noreferrer"><img src="{{ item.image }}" alt="{{ item.alt }}"></a>
+      </div>
+      <div class="col-lg-6 order-md-1">
+        <h4>{{ forloop.index }}: {{ item.title }}</h4>
+        <div class="text-muted">
+          <p>{{ item.desc | markdownify }}</p>
+        </div>
+      </div>
+    </div>
+    {% else %}
+    <div class="row helping align-items-center">
+      <div class="col-lg-6 order-md-1">
+        <a href="{{ item.url }}" target="_blank" rel="noopener noreferrer"><img src="{{ item.image }}" alt="{{ item.alt }}"></a>
+      </div>
+      <div class="col-lg-6 order-md-2">
+        <h4>{{ forloop.index }}: {{ item.title }}</h4>
+        <div class="text-muted">
+          <p>{{ item.desc | markdownify }}</p>
+        </div>
+      </div>
+    </div>
+    {% endif %}
+    {% endfor %}
+  </div>
+</section>
+{% else %}
```

Comment:
There will probably always be a locale, but for consistency, do you want to include the else branch too?